bool status
---
float32 distance
